src/engraving/tests/tst_exchangevoices.cpp

```diff
@@ -135,7 +135,6 @@ void TestExchangevoices::undoChangeVoice()
     score->startCmd();
     score->changeSelectedNotesVoice(1);
     score->endCmd();
-    QSKIP("TODO: Need to confirm beam length difference here");
     QVERIFY(saveCompareScore(score, writeFile1, reference1));
```

Comment:
actually I'd prefer to add some more tests specifically around beaming, and I do have some prepared but they need a bit more work